classes/jailhouse-arch.bbclass

```diff
@@ -0,0 +1,20 @@
+# Set jailhouse architecture JH_ARCH variable
+#
+# return value must match one of architectures supported by jailhouse
+#
+valid_jh_archs = "x86 arm"
```

Comment:
Does `x86` really required in this layer? I guess it would be built for `arm` only since `COMPATIBLE_MACHINE` is set to *mx8m* only...

Reply:
I'm not sure about other boards but most likely all IMX8 will get JH, so we can map on arm64 and get rid of this class, we can safely set JH_ARCH directly in the recipe

Reply:
Cool, I guess the class should be dropped.

Reply:
That's what I'm saying